README.md

```diff
@@ -41,7 +41,7 @@ ReactDOM.render(
 
 This example will render "Hello John" into a container on the page.
 
-You'll notice that we used an HTML-like syntax; [we call it JSX](https://reactjs.org/docs/introducing-jsx.html). JSX is not required to use React, but it makes code more readable, and writing it feels like writing HTML. We recommend using [Babel](https://babeljs.io/) with a [React preset](https://babeljs.io/docs/plugins/preset-react/) to convert JSX into native JavaScript for browsers to digest.
+You'll notice that we used an HTML-like syntax; [we call it JSX](https://reactjs.org/docs/introducing-jsx.html). JSX is not required to use React, but it makes the code more readable and writing it feels like writing HTML. We recommend using [Babel](https://babeljs.io/) with a [React preset](https://babeljs.io/docs/plugins/preset-react/) to convert JSX into native JavaScript for browsers to digest.
```

Comment:
I don't think there's anything wrong with the prior wording. "_it makes code_" is an general statement. "_it makes **the** code_" is more specific. Either _could_ apply in this case, but I think the docs are saying that- in the general case, JSX makes code more readable.

As for the oxford comma you've removed... I'm actually a fan of those. 😄 I think they make sentences more readable.